client/app/components/tags-control/TagsControl.jsx

```diff
@@ -79,14 +79,15 @@ export default class TagsControl extends React.Component {
         <a className="label label-tag" role="none" onClick={this.openEditModal}>
           {buttonLabel}
         </a>
-        {this.state.showModal
-          ? <TagsEditorModal
-            tags={tags}
-            availableTags={this.availableTags}
-            close={this.onTagsChanged}
-            dismiss={this.closeEditModal}
-          />
-          : null
+        {
+          this.state.showModal ? (
```

Comment:
I would change `? () : null` syntax to `&& ()`, but I guess this is not in the scope, so no big deal :slightly_smiling_face:

Reply:
Thanks, fixed :+1: